install/kubernetes/cilium/values.yaml

```diff
@@ -88,31 +88,21 @@ image:
   digest: ""
   useDigest: false
 
-# -- Pod affinity for cilium-agent.
-affinity:
-  nodeAffinity:
```

Comment:
Why did this disappear?

Reply:
TL;DR, using `nodeSelector` instead.
```yaml
nodeSelector:
  kubernetes.io/os: linux
```
I was check, cilium 1.10 support minimal k8s version is 1.16. In that version, all nodes as `kubernetes.io/os` label and `beta.kubernetes.io/os` is deprecated, so that we can simplify long `affinity` config to one line of `nodeSelector`.

Reply:
Ok but we need the `podAntiAffinity` so that we don't have two Cilium pods running on the same host.

Reply:
You can't have 2 pod running in the same host with **DaemonSet** :)

Reply:
@dungdm93 not with the same DaemonSet but you can have 2 Cilium pods running with 2 different DaemonSet and we don't want that.

Reply:
@aanm I thought only one `cilium-agent` DaemonSet per cluster. What's use-case for that?

Reply:
@dungdm93 see https://github.com/cilium/cilium/pull/11830 and the issue that it fixes.

Reply:
@aanm thanks. `podAntiAffinity` is back.
Let take a final look :rocket: